### DISCUSSION TITLE
Refresh edit page header actions from relation

### DISCUSSION THREAD

**Discussion by clinvest** [filamentphp/filament#11480] [Help] [answered]
### Package

Panel builder

### Package Version

v3.0.0

### How can we help you?

Hi all,

Do you know how to refresh the relation parent edit page.

I know this code: 

```#[On('update-program-form')]
    public function refreshForm(): void
    {
        parent::refreshFormData(['name', 'management_status']);
    }
```

But it updates only the form, I need to update the visibility of a header action.

Thank you in advance,

**Comment by leandrocfe** [comment, marked as answer]
I guess you can use [livewire events](https://livewire.laravel.com/docs/events#listening-for-events), registering the event in the EditPage

**Reply by clinvest** [reply]
Yes it was implemented but I wasn't calling the dispatch :( thank you ;)
